lib/test/cdo/test_url_converter.rb

```diff
@@ -15,10 +15,8 @@ def test_no_ops_in_production_configuration
     assert_equal 'https://hourofcode.com', url_converter.replace_origin('https://hourofcode.com')
     assert_equal 'https://csedweek.org', url_converter.replace_origin('https://csedweek.org')
     assert_equal 'https://csedweek.org/about', url_converter.replace_origin('https://csedweek.org/about')
-    assert_equal 'https://uk.code.org', url_converter.replace_origin('https://uk.code.org')
-    assert_equal 'https://uk.code.org/about', url_converter.replace_origin('https://uk.code.org/about')
-    assert_equal 'https://ro.code.org', url_converter.replace_origin('https://ro.code.org')
-    assert_equal 'https://ro.code.org/about', url_converter.replace_origin('https://ro.code.org/about')
+    assert_equal 'https://advocacy.code.org', url_converter.replace_origin('https://advocacy.code.org')
```

Comment:
this change seems unrelated to this PR?

Reply:
Given that we were losing tests on `uk.code.org` and `ro.code.org` I wanted to add a couple additional tests on a subdomain that remained, `advocacy.code.org`.